dm/master/server.go

```diff
@@ -1985,3 +2263,341 @@ func (s *Server) workerArgsExtractor(args ...interface{}) (workerrpc.Client, str
 
 	return cli, worker, nil
 }
+
+// WatchRequest watches requests in etcd, and handle these request, and write response to etcd.
+func (s *Server) WatchRequest(ctx context.Context) {
+	watchCh := s.etcdClient.Watch(context.Background(), defaultOperatePath, -1)
```

Comment:
can we use `ctx` instead of `context.Background()` here? or do we need another timeout context?

Reply:
I use ctx instead, don't need timeout context, because master should not watch request until exit